gem_tasks/cucumber.rake

```diff
@@ -0,0 +1,27 @@
+# Copied from https://github.com/cucumber/cucumber-ruby/blob/master/gem_tasks/cucumber.rake
+
+require 'cucumber/rake/task'
+require 'cucumber/platform'
+
+class Cucumber::Rake::Task
+  def set_profile_for_current_ruby
```

Comment:
`cucumber-rails` does take care but not vanilla `cucumber`.

There are lots of auto generated files out there that are developers responsibility to maintain after they are first generated. Unfortunately cucumber does not. That's why I left the reference up there so we can maintain it ourselves the best we can.

Otherwise, if you are up to maintain, this is a good gem opportunity.

Getting to the point: is cucumber/rake integration important? If so, how does would you do it? Can we get a different PR? What do you suggest?